Makefile

```diff
@@ -144,7 +173,9 @@ verify-envoy-protos:
 # Tests
 #----------------------------------------------------------------------------------
 
-GINKGO_VERSION ?= $(shell echo $(shell go list -m github.com/onsi/ginkgo/v2) | cut -d' ' -f2)
+# GINKGO_VERSION ?= $(shell echo $(shell go list -m github.com/onsi/ginkgo/v2) | cut -d' ' -f2)
+# TEMP: Hardcoding version for now, to match go.mod
+GINKGO_VERSION ?= v2.19.0
```

Comment:
Can we undo these temp changes and continue to rely on the go.mod definition to capture the ginkgo version?

Reply:
great idea